vortex-layout/src/scan/tree_row_mask.rs

```diff
@@ -0,0 +1,281 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+use std::ops::RangeInclusive;
+use std::sync::Arc;
+
+use roaring::RoaringTreemap;
+
+/// Uses a roaring treemap to store a set of row indices.
+/// This can be refined and a row range query can be performed.
+#[derive(Debug, Clone)]
+pub struct TreeRowMask {
```

Comment:
I don't know that it's worth creating this struct, given it's basically trivial to construct a `RoaringTreemap::from_range`?

Reply:
I didn't find that. A full treemap is quite large